app/src/sharedTest/java/org/oppia/android/app/devoptions/featureflags/FeatureFlagsFragmentTest.kt

```diff
@@ -1246,6 +1240,102 @@ class FeatureFlagsFragmentTest {
     }
   }
 
+  @Test
+  fun testFeatureFlagsFragment__clickSave_showsRestartDialog_configChange_dialogPersists() {
```

Comment:
`testFeatureFlagsFragment_`
```suggestion
  fun testFeatureFlagsFragment_clickSave_showsRestartDialog_configChange_dialogPersists() {
```

Reply:
Updated